console/console.properties

```diff
@@ -264,7 +264,7 @@ AreasGroup=INSEE_DEP
 #subject.new.account.notification=[${instanceName}] New account created
 
 # Encoding of the email templates
-# This "�" char should display nicely in a ISO 8859-1 configured editor
+# This "�" char should display nicely in a ISO 8859-1 configured editor
```

Comment:
Still a "é" char ?

Reply:
extra whitespace at the end of the line being removed